packages/@aws-cdk/aws-cognito/lib/user-pool.ts

```diff
@@ -901,9 +871,36 @@ export class UserPool extends UserPoolBase {
     }
     return schema;
   }
+
+  private renderAttribute(
```

Comment:
I split it to make it more slim and readable but since now I'm also "rendering" the standard attrs I think I can merge it back, so then we have it all together